include/megaapi.h

```diff
@@ -18523,6 +18525,43 @@ class MegaApi
          */
         void queryGoogleAds(int adFlags, MegaHandle publicHandle = INVALID_HANDLE, MegaRequestListener *listener = nullptr);
 
+        /**
+         * @brief Set a bitmap to indicate whether some cookies are enabled or not
+         *
+         * The associated request type with this request is MegaRequest::TYPE_SET_COOKIE_SETTINGS
```

Comment:
Incorrect: `TYPE_SET_ATTR_USER`
```suggestion
         * The associated request type with this request is MegaRequest::TYPE_SET_ATTR_USER
```